web_accounts_list.json

```diff
@@ -4591,6 +4591,73 @@
            "known_accounts" : ["john", "junech"],
            "category" : "dating",
            "valid" : true
+          },
+          {
+           "name" : "icq-chat",
+           "check_uri" : "https://icq.icqchat.co/members/{account}/",
+           "account_existence_code" : "200",
+           "account_existence_string" : "ICQ chat",
+           "account_missing_string" : "Oops! We ran into some problems",
+           "account_missing_code" : "404",
+           "known_accounts" : ["brookenora.54", "bigdaddy.77"],
+           "category" : "social",
+           "valid" : true
+          },
+          {
+           "name" : "HeyLink.me",
+           "check_uri" : "https://heylink.me/{account}/",
+           "account_existence_code" : "200",
+           "account_existence_string" : "Heylink.me",
+           "account_missing_string" : "We can't find the page that you're looking for :(",
+           "account_missing_code" : "404",
+           "known_accounts" : ["mohammed13", "johnny"],
+           "category" : "misc",
+           "valid" : true
+          },
+          {
+           "name" : "Yazawaj",
+           "check_uri" : "https://www.yazawaj.com/profile/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "أنا",
+           "account_missing_string" : "الصفحة المطلوبة غير موجودة",
+           "account_missing_code" : "404",
+           "known_accounts" : ["monya14555d", "LordMohy"],
+           "category" : "dating",
+           "valid" : true
+          },
+          {
+           "name" : "Alloannonces",
+           "check_uri" : "https://www.alloannonces.ma/{account}/",
+           "account_existence_code" : "200",
+           "account_existence_string" : "Vendeurs/Agents",
+           "account_missing_string" : "Page non defini",
+           "account_missing_code" : "404",
+           "known_accounts" : ["Rachid13", "Ayoub"],
+           "category" : "social",
+           "valid" : true
+          },
+          {
+           "name" : "contactos.sex",
+           "check_uri" : "https://www.contactossex.com/profile/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "Información Personal",
+           "account_missing_string" : "Desde 2001 conectando gente!",
+           "account_missing_code" : "200",

```

Comment:
I think you are testing these sites with a tool that follows redirects instead of showing them to you. For me, when I visit a non-existing username like `https://www.contactossex.com/profile/FYLydPyK1E67YD46EQ2U` the site redirects my browser with an HTTP 302 code instead of the 200 you have here. Can you test that more?

Reply:
Tested with an online tool [httpstatus ](https://httpstatus.io), got a 302 also, **will amend**.